schemas/kb.py

```diff
@@ -0,0 +1,81 @@
+"""
+Knowledge Base Schema
+
+This is a very general schema that covers many information extraction-style
+tasks, including:
+
+- Named entity recognition (NER)
+- Named entity disambiguation/canonicalization/normalization (NED)
+- Event extraction
+- Relation extraction (RE)
+- Coreference resolution
+
+This schema assumes a document with child elements (e.g., entities, relations)
+and a flat hierarchy of document passages.
+
+"""
+import datasets
+
+
+features = datasets.Features(
+    {
+        "id": datasets.Value("string"),
+        "document_id": datasets.Value("string"),
+        "passages": datasets.Sequence(
+            {
+                "id": datasets.Value("string"),
+                "type": datasets.Value("string"),
+                "text": datasets.Value("string"),
+                "offsets": datasets.Sequence([datasets.Value("int32")]),
```

Comment:
if we're not allowing for disjoint passages I think we should have just `Sequence(Value("int32"))` here  (OR if we are, then make the `text` a sequence as well). I think we don't need disjoint passages.